src/app/build.json

```diff
@@ -11,5 +11,6 @@
   "fileIssueUrl": "https://github.com/cryostatio/cryostat/issues/new?labels=user+report,bug&body=Affects+__REPLACE_VERSION__",
   "mailingListName": "Google Groups",
   "mailingListUrl": "https://groups.google.com/g/cryostat-development",
-  "licenseUrl": "https://github.com/cryostatio/cryostat/blob/main/LICENSE"
+  "licenseUrl": "https://github.com/cryostatio/cryostat/blob/main/LICENSE",
+  "version": "2.4.0-dev"
```

Comment:
We'll need to add a step to our release documentation that this key needs to be updated.

Reply:
Yep! Updated our Upstream Release Process Docs now :))